.packit.yaml

```diff
@@ -0,0 +1,45 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-smart_proxy_pulp.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-smart_proxy_pulp.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: smart_proxy_pulp
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-smart_proxy_pulp
+
+upstream_tag_template: '{version}'
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec -O rubygem-smart_proxy_pulp.spec"
+  get-current-version:
+    - ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("*.gemspec").first).version'
+  create-archive:
+    - gem build smart_proxy_pulp.gemspec
+    - bash -c "ls -1t ./smart_proxy_pulp-*.gem | head -n 1"
+
+jobs:
+  - job: copr_build
+    trigger: pull_request
+    targets:
+      rhel-8:
```

Comment:
We're about to drop el8. This reminds me we need update all packit configs as well